# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/webcodecs/video-decoder.https.any.js.ini
# Commit: 02cb31bfdc0f
# Full Hash: 02cb31bfdc0f4a15397b3a4e0725c408d738b34c
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2023-06-30 04:06:43
# Regressor Bug: 1831451
# File Overlap Count: 1
# Description:
#   Bug 1831451 - Implement VideoDecoder::IsConfigSupported r=padenot
#   
#   This patch implements the isConfigSupported method for WebCodecs'
#   VideoDecoder.
#   
# ==============================================================================

diff -r 77b0f5226ed8 -r 02cb31bfdc0f testing/web-platform/meta/webcodecs/video-decoder.https.any.js.ini
--- a/testing/web-platform/meta/webcodecs/video-decoder.https.any.js.ini	Thu Jun 29 22:33:11 2023 +0000
+++ b/testing/web-platform/meta/webcodecs/video-decoder.https.any.js.ini	Thu Jun 29 22:33:11 2023 +0000
@@ -1,20 +1,7 @@
 [video-decoder.https.any.html]
+  prefs: [dom.media.webcodecs.enabled:true]
   expected:
     if (os == "android") and fission: [OK, TIMEOUT]
-  [Test that VideoDecoder.isConfigSupported() rejects invalid config:Empty codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.isConfigSupported() rejects invalid config:Unrecognized codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.isConfigSupported() rejects invalid config:Audio codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.isConfigSupported() rejects invalid config:Ambiguous codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.isConfigSupported() rejects invalid config:Codec with MIME type]
-    expected: FAIL
 
   [Test that VideoDecoder.configure() rejects invalid config:Empty codec]
     expected: FAIL
@@ -36,22 +23,9 @@
 
 
 [video-decoder.https.any.worker.html]
+  prefs: [dom.media.webcodecs.enabled:true]
   expected:
     if (os == "android") and fission: [OK, TIMEOUT]
-  [Test that VideoDecoder.isConfigSupported() rejects invalid config:Empty codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.isConfigSupported() rejects invalid config:Unrecognized codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.isConfigSupported() rejects invalid config:Audio codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.isConfigSupported() rejects invalid config:Ambiguous codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.isConfigSupported() rejects invalid config:Codec with MIME type]
-    expected: FAIL
 
   [Test that VideoDecoder.configure() rejects invalid config:Empty codec]
     expected: FAIL